CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.0.0-rc.0 (2020-02-18)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#7](https://github.com/blackbaud/skyux-omnibar-interop/pull/7)
```

Comment:
Just to sanity check: Are we pointing to the right PR? I assume so since it deals with Angular 9. But I don't see anything about Angular Package Format or transpiling in the code changes, so I just want to make sure.

Reply:
We are. The Angular Package Format comes from the `4.0.0-rc.0` version of Builder.